### ISSUE TITLE
Restart the ECU (soft reset) over GUI

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#18]
## User story
1. As a user
2. I want to initiate a soft reset of the ECU from the GUI
3. So that I can restart the ECU without direct access to the hardware.

## Acceptance criteria
* The GUI includes a button or option to initiate the soft reset of the ECU
* Message to initiate the soft reset is sent
* Upon successful reset, the GUI provides feedback to the user indicating that the ECU has been restarted
* If dependencies are implemented: The 'resetECU()' method in the SessionManager class is called when the user triggers the soft reset action from the GUI

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
